site/app/controllers/PollController.php

```diff
@@ -188,6 +188,25 @@ public function endPoll() {
         );
     }
 
+    /**
+     * @Route("/courses/{_semester}/{_course}/polls/setClosed", methods={"POST"})
+     * @AccessControl(role="INSTRUCTOR")
+     * @return MultiResponse
+     */
+    public function closePoll() {
+        if (!isset($_POST["poll_id"])) {
+            $this->core->addErrorMessage("Invalid Poll ID");
+            return MultiResponse::RedirectOnlyResponse(
+                new RedirectResponse($this->core->buildCourseUrl(['polls']))
```

Comment:
If you're sending back only one type of response, you can just do 
```php
return new RedirectResponse( ...
```
Remember to edit the phpdocstring `@return` or remove it, otherwise phpstan will complain

Reply:
@BenKelly99  did that for most functions in PollController.php so I'm scared to change them all in case it was intended to be like that

Reply:
That's fine, we can refactor the older code later as long as the new stuff we're adding matches our up to date functions

Reply:
Yeah when I made this it was right when a lot of changes were made to the responses. I can refactor it later